test/synchronicity_test.py

```diff
@@ -66,6 +66,32 @@ async def test_function_async():
     assert await coro == 1764
 
 
+@pytest.mark.asyncio
+async def test_function_async_as_function_attribute():
+    s = Synchronizer()
+    t0 = time.time()
+    f_s = s.create_blocking(f).aio
```

Comment:
instead of `create_blocking` maybe we should just call it `create`?

Reply:
We already have a deprecated `.create()` which returns:
```
{
            Interface.ASYNC: self.create_async(obj),
            Interface.BLOCKING: self.create_blocking(obj),
        }
```
in order to support old modal clients that didn't pin the synchronicity version. We'd have to wait until we phase out those modal versions until we reuse .create for something else right?

Reply:
We have had the `"Using latest synchronicity with an old interface - please upgrade to latest modal-client!"` warning there though, so perhaps it's acceptable to phase out now?

Reply:
Yeah seems fine to phase out!

Reply:
PR here: https://github.com/modal-labs/synchronicity/pull/100